Yoast/wordpress-seo
# Style Metabox menu padding

## Summary

This PR can be summarized in the following changelog entry:

* Add style for padding to metabox menu to avoid being overwritten by custom editor styles.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Add custom editor style which includes "padding: 1.5rem".
* Go to post edit screen and view Yoast metabox. 
* See horizontal tabbed menu has correct alignment.

## UI changes

## Documentation

## Quality assurance


Fixes #11380
